src/main/java/org/galatea/starter/service/PriceService.java

```diff
@@ -6,37 +6,48 @@
 import net.sf.aspect4log.Log;
 import org.springframework.stereotype.Service;
 
+
 @RequiredArgsConstructor
 @Slf4j
 @Log
 @Service
-public class HalService {
+public class PriceService {
 
   /* We need to have this constant so sonar qube doesn't complain */
   private static final String DERP = "derp!";
+//  public static String stock;
+//  public static String daysToLookBack;
 
   /**
-   * Process the text from GET command into the appropriate command
+   * Process the stock from GET command into the appropriate command
    *
-   * @param text the full text from the GET command. Wit.ai will break this down
+   * @param stock from the full text from the GET command. Wit.ai will break this down
+   * @param daysToLookBack from the full text from the GET command. Wit.ai will break this down
    * @return the result of executing the command with the given parameters
    */
-  public String processText(String text) {
-    switch (text) {
-      case "coin-flip":
-        return coinFlip();
-      case "derp":
-        return getDerp();
-      default:
-        return "Unsupported command";
-    }
+
+
+  public String processStock(String stock, String daysToLookBack) {
+
+//    int daysToLookBack2 = Integer.parseInt(daysToLookBack);
```

Comment:
for now have this method return the parameters concatenated into a string somehow